ui/site/src/component/sound.ts

```diff
@@ -67,7 +56,7 @@ const sound: SoundI = new (class {
     }
     const s = this.getOrLoadSound(name, set);
 
-    const doPlay = () => s.volume(this.getVolume() * (volume || 1)).play();
+    const doPlay = () => (s.volume(this.getVolume() * (volume || 1)) as Howl).play();
```

Comment:
If anyone has a suggestion on how to improve this, do let me know because when you only send one number parameter it has the type to give `Howl | number` - It'll give `Howl` if the param is `0..1`.